storage/storage-mq/src/storage-content/postgresStorageContentRepository.ts

```diff
@@ -79,7 +79,17 @@ export class PostgresStorageContentRepository implements StorageContentRepositor
       if (!tableExists) {
         throw new Error(`Table "${tableIdentifier}" does not exist - can not save content`)
       }
-      const values = [content.data, content.pipelineId, content.timestamp]
+
+      /**
+       * When passed an array as value, pg assumes the value is meant to be a native Postgres array
+       * and therefore fails with a "invalid input syntax for type json" error when the target field
+       * is actually of type jsob/jsonb.
+       *
+       * Ref: https://github.com/brianc/node-postgres/issues/2012
+       */
+      const data = Array.isArray(content.data) ? JSON.stringify(content.data) : content.data
```

Comment:
Other options to this very local fix include overwriting the global transformation function in pg's utils.js or adding this logic to your own node-dry-pg. I decided against both and a very local fix here because I am missing the big picture for now.